keps/135-startup-policy/README.md

```diff
@@ -0,0 +1,315 @@
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+This KEP aims to add `spec.startupPolicy` filed to LeaderWorkerSet's API, to indicate the startup order of leader statefulset and worker statefulsets.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+Some frameworks require the leader statefulset in LeaderWorkerSet to be ready (like ray head) before the worker statefulsets are started so they can connect to it.
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+- Add a new API `WaitForLeaderReady` to indicate if we need to create worker statefulsets until the leader statefulset is ready.
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+We propose a new field `spec.startupPolicy` to the LeaderWorkerSet API. This field is defaulted to `Default` which means that lws controller will create worker statefulsets immediately after leader statefulset created. It doesn't guarantee the order of readiness for leader statefulset and worker statefulsets.
+Another option for this field is `WaitForLeaderReady`. It will delay creating the worker statefulsets until the leader statefulset is ready.
```

Comment:
```suggestion
Another option for this field is `WaitForLeaderReady`. It will delay creating the worker statefulset until its leader pod is ready.
```